update_credentials.sh

```diff
@@ -68,4 +76,4 @@ else
     exit
 fi
 
-echo "... DONE updating GeoServer Credentials ..."
+echo "... DONE updating GeoServer Credentials ..."
```

Comment:
```suggestion
echo "... DONE updating GeoServer Credentials ..."

```

now a newline here please :-)